Makefile

```diff
@@ -38,14 +57,27 @@ subsetData: $(SUBSET_DATA_OFILES)
 
 clean:
 	rm -f $(ESTIMATE_OFILES) $(SIPNET_OFILES) $(TRANSPOSE_OFILES) $(SUBSET_DATA_OFILES) estimate sensTest  sipnet transpose subsetData
-
+	rm -rf $(DOXYGEN_HTML_DIR) $(DOXYGEN_LATEX_DIR)
 #clean:
 #	rm -f $(ESTIMATE_OFILES) $(SENSTEST_OFILES) $(SIPNET_OFILES) $(TRANSPOSE_OFILES) $(SUBSET_DATA_OFILES) estimate sensTest  sipnet transpose subsetData
 
+help:
+	@echo "Available targets:"
+	@echo "  help        - Display this help message."
+	@echo "  === core targets ==="
+	@echo "  all         - Builds all components."
+	@echo "  document    - Generate documentation."
+	@echo "  sipnet      - Builds the 'sipnet' executable."
+	@echo "  clean       - Removes compiled files and executables."
```

Comment:
Add documentation to the list here:

	@echo "  clean       - Removes compiled files, executables, and documentation."